lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -470,9 +482,18 @@ inlineInputOnly(hw::HWModuleOp oldMod, hw::InstanceGraph &instanceGraph,
           instParentNode->addInstance(innerInst, innerInstModule);
         }
 
-        // If the op has an inner sym, then attach an updated inner sym.
-        if (auto innerSym = op.getAttrOfType<StringAttr>("inner_sym"))
-          clonedOp->setAttr("inner_sym", symMapping[innerSym]);
+        // If the cloned op has an inner sym, then attach an updated inner sym.
+        if (auto innerSymOp = dyn_cast<hw::InnerSymbolOpInterface>(clonedOp)) {
+          SmallVector<hw::InnerSymPropertiesAttr> properties;
+          for (auto property : innerSymOp.getInnerSymAttr()) {
```

Comment:
Check for innerSymAttr != null? Or is that guaranteed by something else? If the latter, I would add an assert to that effect.

Reply:
Yeah, should definitely be checking here.  It sort-of always works since the only operation we could be referring to was an `instanceOp` that was bound in (and must therefore have a symbol).  But still a bug, and a landmine for when someone allows inlining for other operations which might not have symbols.